packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRefreshSObjects.test.ts

```diff
@@ -51,10 +50,7 @@ describe('ForceGenerateFauxClasses', () => {
 
     const projectPath = path.join('sample', 'path');
     const sobjectsPath = path.join(
-      projectPath,
-      SFDX_DIR,
-      TOOLS_DIR,
-      SOBJECTS_DIR
+      projectPaths.toolsFolder(), SOBJECTS_DIR
```

Comment:
Maybe create a const outside and use it here and on line 116?

Reply:
done